### DISCUSSION TITLE
Automated FastDL script cs1.6 server

### DISCUSSION THREAD

**Discussion by amine-aziz** [OpenGamePanel/OGP-Website#622] [General]
**I have made an automated FastDL script which is used for cs1.6 server**

**Requirement :** 

OpenGamePanel
FastDownload Module from OGP

the script is automated

https://github.com/Amine-Aziz/AutoFastdlcs1.6_forOGP/

**Comment by Zorrototo** [comment]
You will have no visibility here, probably should share on the forum for others to see. Probably also explain how to install and have it to work.

What is '$a' in your script?

**Reply by amine-aziz** [reply]
I have already share it in forum and i explain it in GitHub.

the '$a' it's for write the fastdl cvar configurations in the end of server.cfg file
